app/services/licences/fetch-licence-version-purpose-condition.service.js

```diff
@@ -10,45 +10,74 @@ const LicenceVersionPurposeCondtionModel = require('../../models/licence-version
 /**
  * Fetch the matching licence version purpose conditions and return data needed for the view licence page
  *
- * @param {string} id The UUID for the licence_version_purpose_id to fetch
+ * @param {object} ids The object contains the UUIDs for the licence_version_purpose_id and the purpose_id
  *
  * @returns {Promise<Object>} the data needed to populate the view licence page and some elements of the summary tab
  */
-async function go (id) {
-  if (!id) {
+async function go (ids) {
```

Comment:
I think if this was just an array of UUIDs i.e. strings then ids would be ok (I'd probably still try and come up with some more descriptive 😛)

But we give it a more descriptive name when it finally gets to the function that uses it; `licenceVersionPurposeAndPurposeUseIds`. So, I'd suggest renaming the param to that 🤷